app/assets/stylesheets/manageiq-v2v/type-ahead-select.scss

```diff
@@ -0,0 +1,61 @@
+/*
+  Until we can properly import patternfly-react CSS extensions
+  (which will probably require changes to manageiq-ui-classic's webpack config)
```

Comment:
What's the problem there @mturley ? Do we have an issue?

Reply:
Yes, https://github.com/patternfly/patternfly-react/issues/673 is tracking the upstream issue. Importing the patternfly-react CSS directly results in a relative import of the Typeahead.css file which doesn't exist, when it should instead be inlined. I made a PR to fix that issue upstream, I'll keep an eye on it and remove this copied CSS when it's resolved.

I was initially thinking we the import was failing due to something in the ui-classic webpack config, but I was wrong, I'll remove that part of this comment.

Reply:
Aah, thanks :) Yeah, I saw that issue but was confused with the ui-classic mention. Thanks :)